test/compare-nr.test.js

```diff
@@ -0,0 +1,18 @@
+test('compareNr 1.0', async () => {
+  const equals = await $app['compareNr 1.0']({ leftValue: 100, rightValue: 100, comparator: 'eq' });
+  const nEquals = await $app['compareNr 1.0']({ leftValue: 100, rightValue: 0, comparator: 'neq'});
+  const lower = await $app['compareNr 1.0']({ leftValue: 100, rightValue: 50, comparator: 'lower'});
+  const lowerEq = await $app['compareNr 1.0']({ leftValue: 100, rightValue: 99, comparator: 'lowerEq'});
+  const higher = await $app['compareNr 1.0']({ leftValue: 200, rightValue: 100, comparator: 'higher'});
+  const higherEq = await $app['compareNr 1.0']({ leftValue: 100, rightValue: 100, comparator: 'higherEq'});
+
+  
+  assert(equals, { result: true });
+  assert(nEquals, { result: true });
+  assert(lower, { result: false });
+  assert(lowerEq, { result: false });
+  assert(higher, { result: true });
+  assert(higherEq, { result: true });
+
+
+})
```

Comment:
I think it's better to split this in multiple tests, one per comparator. So something like 
```javascript
test('Using "eq" returns true if the values are equal', async () => {
  const { result } = await $app['compareNumber 1.0']({ leftValue: 1, rightValue: 1, comparator: 'eq' });

  assert(result, true);
});

test('Using "eq" returns false if the values are not equal', async () => {
  const { result } = await $app['compareNumber 1.0']({ leftValue: 1, rightValue: 2, comparator: 'eq' });

  assert(result, false);
});
```